.github/workflows/gradle-cache-check.yml

```diff
@@ -0,0 +1,31 @@
+name: Periodic Gradle Build Cache Check
+on:
+  schedule:
+    - cron: '0 01 * * 1'
+
+env:
+  LC_ALL: "en_US.UTF-8"
+  GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
+
+jobs:
+  build-cache-check:
+    if: github.repository == 'line/armeria'
+    runs-on: self-hosted
+    steps:
+      - id: setup-jdk-19
+        name: Set up JDK 19
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '19'
+
+      - uses: gradle/gradle-enterprise-build-validation-scripts/.github/actions/gradle/download@actions-stable
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: gradle/gradle-enterprise-build-validation-scripts/.github/actions/gradle/experiment-3@actions-stable
+        with:
+          gitRepo: https://github.com/line/armeria.git
+          gitBranch: main
+          tasks: build
+          args: --parallel -Pretry=true -PfailOnPassedAfterRetry=false
```

Comment:
I'd suggest you add the `failIfNotFullyCacheable` argument to the action used here to fail the whole build if it's not fully cacheable, meaning that it fails if any cacheable tasks don't get their second run result from cache.

Reply:
Isn't the `-f` flag added if not specified by default anyways?

https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/b5e54b97633dd851bb59b135538c722a01c7c5db/.github/actions/gradle/experiment-3/action.yml#L105C13-L105C44